cornflow-server/cornflow/shared/authentication.py

```diff
@@ -3,6 +3,7 @@
 """
 
 # Global imports
+import logging as log
```

Comment:
Instead of logging it should be:

```python
form flask import current_app

```
And then:

``` python
current_app.logger.error(f"The permission for this endpoint is not in the database: {url}")
```
or 
```python
raise NoPermission(error="You do not have permission t access this endpoint", log_txt=f"The permission for this endpoint is not in the database: {url}", status_code=403)
```